### ISSUE TITLE
tmux doesn't reattach properly

### ISSUE THREAD

**Issue by bluddy** [tmux/tmux#276]
I sometimes accidentally close my iterm window on OSX. Not a big deal -- that's what I'm running tmux for, right? Except that recently I do a tmux attach, and I can scroll through the tmux windows, but I can't enter any input in any of them! Any keyboard input just gets completely ignored. Any idea what this could be?

**Comment by zackp30** [comment]
I remember commenting on a bug about something along the lines of what you're describing. It can also be triggered when an SSH connection remains attached to the tmux session, after closing the main terminal window (whether it be Konsole running `ssh`, or just plain PUTTY), anyway, try using <kbd>Ctrl-b Shift-d</kbd> which will bring up a list of sessions attached to the tmux server, using one of the arrow keys (the one that issued the command to list sessions is already selected, so move to any other one), then press enter.

I guess this is an issue with tmux trying to use a dead PTY but obviously failing.

TL;DR: Detach the dead sessions using `tmux list-sessions` or <kbd>Ctrl-b Shift-d</kbd>.

This worked for me.
